api/data/NMBS/connections.php

```diff
@@ -421,14 +422,14 @@ private static function addOccupancy($connections, $date)
                 if (!is_null($connection->via)) {
                     foreach ($connection->via as $key => $via) {
                         if ($key < count($connection->via) - 1) {
-                            $vehicle = substr(strrchr($connection->via[$key + 1]->vehicle, "."), 1);
+                            $vehicleURI = 'http://api.irail.be/vehicle/?id=' . $connection->via[$key + 1]->vehicle;
                         } else {
-                            $vehicle = substr(strrchr($connection->arrival->vehicle, "."), 1);
+                            $vehicleURI = 'http://api.irail.be/vehicle/?id=' . $connection->arrival->vehicle;
```

Comment:
where did you get `?id=` from? I would use `http://api.irail.be/vehicle/` as the base URI only... Still need to create a page for that though!